include/vcpkg/sourceparagraph.h

```diff
@@ -209,6 +210,10 @@ namespace vcpkg
         SchemedVersion schemed_version() const { return {scheme(), to_version()}; }
         VersionSpec to_version_spec() const { return source_control_file->to_version_spec(); }
         Path port_directory() const { return control_path.parent_path(); }
+        bool is_overlay_port(const Path& default_port_dir) const
```

Comment:
This is not correct. For example, ports loaded from a registry won't be in the default port directory, but are not overlay ports.

(I agree it looks like we might be currently printing an incorrect message)